server/public/model/role.go

```diff
@@ -109,7 +109,6 @@ func init() {
 			PermissionGetAnalytics,
 		},
 		PermissionSysconsoleReadReportingTeamStatistics.Id: {
-			PermissionViewTeam,
```

Comment:
Any chance we could add a unit test as well? e2e looks excellent, but I'd prefer it not be a replacement for security-sensitive functionality, especially as we skip them so often that I'd feel better to have both :)

Reply:
I originally started with the unit test (it was easier!) but I discovered that the system console works by first fetching the ancillary permissions for a given system console permission, then figuring out the new set of permissions, and then POSTing that as the final set of permissions for a given role. (🤯)

Since only the E2E test directly running this client-side logic actually exercised the semantics under test, I figured the unit test wouldn't be meaningful. Thoughts?

Reply:
Interesting. Would the client logic be replicable using `model.Client4` methods? Then, perhaps, something in `api4` would make sense.

Reply:
Yes, that's what the unit test did... but it amounted to testing the ancillary permissions fetch followed by the permissions set (both of which are already tested).

If the logic in the system console changes, the unit test wouldn't fail. Do you think this is a valuable addition? (It's easy to generate!)

Reply:
For me, the unit test is valuable as long as it would fail without the changes in here.

Reply:
Agreed. In that case, I'll skip it here since I can't test the client-side behaviour from the server alone.